app/assets/stylesheets/application.css.scss

```diff
@@ -80,6 +80,32 @@ input {
  margin-left: -17px;
 }
 
+.clearfix {
+  overflow: auto;
+}
+
+#results {
+  padding: 10px;
+  background: rgba(0, 0, 0, 0.3);
+  border-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.